### PR TITLE
[IMP] purchase_last_price_info

### PR DESCRIPTION
**Pull request by ageficent**
Improvement of module purchase_last_price_info

Features added:
* Update readme format according to OCA rules
* Add fields to product template form and tree view
* Add fields to product product tree view

cc ~ @Eficent

### CODE REVIEW

File: purchase_last_price_info/views/product_views.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	</record>
+
+
+	<record id="product_product_tree_view" model="ir.ui.view">
+		<field name="name">product.product.tree</field>
+		<field name="model">product.product</field>
+		<field name="inherit_id" ref="product.product_product_tree_view" />
+		<field name="arch" type="xml">
+			<field name="price" position="after">
+				<field name="last_purchase_price"/>
```
Comment (celm1990) [inline purchase_last_price_info/views/product_views.xml]: Field is not stored on DB, think is bad idea show on tree view

File: purchase_last_price_info/views/product_views.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		</field>
+	</record>
+
+	<record id="product_template_tree_view" model="ir.ui.view">
+        <field name="name">product.template.product.tree</field>
+        <field name="model">product.template</field>
+		<field name="inherit_id" ref="product.product_template_tree_view"/>
+        <field name="arch" type="xml">
+			<field name="standard_price" position="after">
+				<field name="last_purchase_price"/>
```
Comment (celm1990) [inline purchase_last_price_info/views/product_views.xml]: Same Here

**Comment by celm1990** [comment]
@ageficent  PR must be make to OCA's repository

**Comment by ageficent** [comment]
> @ageficent PR must be make to OCA's repository

That does not make sense, if I do a PR to the OCA's repository, then there would be two PR for the migration of this module and one would need to be closed. That is why I did it in your branch so your Pull Request is kept but with my proposed changes are included as well.

**Comment by celm1990** [comment]
I can close my PR on OCA, is not problem.

**Comment by ageficent** [comment]
> I can close my PR on OCA, is not problem.

Okay, I will try to find a workaround solution to remove the computed field in the tree view and do a Pull Request in the OCA repo :+1:

**Pull request closed without merging**